modules/ding_content_rss/views/ding_content_rss_handler_library_id.inc

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * @file
+ * Node's "Library ID" custom field handler.
+ */
+
+/**
+ * Class ding_content_rss_handler_library_id.
+ */
+// @codingStandardsIgnoreStart
+class ding_content_rss_handler_library_id extends views_handler_field {
+
+  /**
+   * {@inheritdoc}
+   */
+  function option_definition() {
```

Comment:
This function don't seam to do anything but call the parent object, so it could be omitted.